examples/all.py

```diff
@@ -171,6 +171,8 @@ def run_example(example, reporter=None):
         else:
             mod.main()
         return True
+    except KeyboardInterrupt:
+        sys.exit(0)
```

Comment:
I'm assuming you are doing this so that it doesn't say "fail". I think it should at least have `raise` here instead of `sys.exit(0)`. You might want to interrupt a slow example and see a traceback.